orchestration/dap_orchestration/repositories/prod_repositories.py

```diff
@@ -52,13 +57,16 @@ def weekly_sample_refresh(context: ScheduleEvaluationContext) -> dict[str, objec
 def repositories() -> list[PipelineDefinition]:
     return [
         weekly_sample_refresh,
-        refresh_data_all.to_job(resource_defs={
-            "extract_beam_runner": preconfigure_resource_for_mode(k8s_dataflow_beam_runner, "prod_extract"),
-            "transform_beam_runner": preconfigure_resource_for_mode(k8s_dataflow_beam_runner, "prod_transform"),
-            "refresh_directory": refresh_directory,
-            "outfiles_writer": outfiles_writer,
-            "api_token": preconfigure_resource_for_mode(api_token, "prod"),
-            "io_manager": preconfigure_resource_for_mode(gcs_pickle_io_manager, "prod"),
-            "gcs": google_storage_client,
-        })
+        refresh_data_all.to_job(
+            resource_defs={
+                "extract_beam_runner": preconfigure_resource_for_mode(k8s_dataflow_beam_runner, "prod_extract"),
+                "transform_beam_runner": preconfigure_resource_for_mode(k8s_dataflow_beam_runner, "prod_transform"),
+                "refresh_directory": refresh_directory,
+                "outfiles_writer": outfiles_writer,
+                "api_token": preconfigure_resource_for_mode(api_token, "prod"),
+                "io_manager": preconfigure_resource_for_mode(gcs_pickle_io_manager, "prod"),
+                "gcs": google_storage_client,
+            },
+            executor_def=in_process_executor
```

Comment:
⛏️ A comment here explaining why we're using the in_process_executor would be helpful for future debugging.